source/index.txt

```diff
@@ -23,14 +23,20 @@ Introduction
 ------------
 
 Welcome to the documentation site for the Kotlin Driver, the official
-MongoDB driver for server-side Kotlin applications using coroutines. Download it using
-`Maven <https://maven.apache.org/>`__ or
-`Gradle <https://gradle.org/>`__, or set up a runnable project by
-following our Quick Start guide.
-
-If your Kotlin application requires synchronous processing, use the
-:driver:`Sync Driver </kotlin-sync/>` which uses synchronous operations
-to make blocking calls to MongoDB.
+MongoDB driver for server-side Kotlin applications by using coroutines.
+Download the driver by using `Maven <https://maven.apache.org/>`__ or `Gradle
+<https://gradle.org/>`__, or set up a runnable project by following our
+Quick Start guide.
+
+.. tip:: Other Kotlin Platforms for MongoDB
+
+   If your Kotlin application requires synchronous processing, use the
+   :driver:`Sync Driver </kotlin-sync/>`, which uses synchronous operations
+   to make blocking calls to MongoDB.
+   
+   If you are developing an Android or Multiplatform application, you can
```

Comment:
S: lowercase multiplatform (unless there's a specific reason it's capitalized)

```suggestion
   If you are developing an Android or multiplatform application, you can
```

Reply:
https://kotlinlang.org/docs/multiplatform.html based on this, it seems like Kotlin Multiplatform (KMP) is a proper noun. Making this more clear